redbeams-api/src/main/java/com/sequenceiq/redbeams/doc/ModelDescriptions.java

```diff
@@ -3,27 +3,26 @@
 public final class ModelDescriptions {
 
     public static final String ID = "ID of the resource";
+    public static final String CRN = "CRN of the resource";
+    public static final String CREATED = "creation time of the resource in long";
```

Comment:
nit: It seems that the convention here is to capitalize the first letter, here and below.

Reply:
Actually - the CREATION_DATE constant below here (line 9) should be used instead, since it means the same thing.

Reply:
done, thanks!